code/__DEFINES/subsystems.dm

```diff
@@ -152,6 +152,7 @@
 #define INIT_ORDER_TICKER 55
 #define INIT_ORDER_TCG 55
 #define INIT_ORDER_MAPPING 50
+#define INIT_ORDER_TTS 49 // Needs to be before Early Assets init
```

Comment:
Why? 

Early assets only exists for CI purposes. It's so that we can make sure preferences menu doesn't runtime if it's loaded before the atoms SS.

In the real game, preferences assets can be loaded at basically *any* time, because the moment you open up the preferences menu, they all generate.

Which means I do not think whatever you're trying to guarantee here actually works.